experiment/simulation/js/animation.js

```diff
@@ -0,0 +1,494 @@
+}
+
+
+// function to disappear the output text
+function outputDisappear() {
+    for (const text of TEXTOUTPUT) {
+        objectDisappear(text);
+    }
+}
+// function to appear the input text
+function outputVisible() {
+    for (const text of TEXTOUTPUT) {
+        objectAppear(text);
+    }
+}
+function xTextDisappear() {
+    objectDisappear(TEXTINPUT[0]);
+}
+function xTextAppear() {
+    objectAppear(TEXTINPUT[0]);
+}
+function yTextDisappear() {
+    objectDisappear(TEXTINPUT[1]);
+}
+function yTextAppear() {
+    objectAppear(TEXTINPUT[1]);
+}
+function clockDisappear() {
+    objectDisappear(TEXTCLOCK[0]);
+}
+function clockAppear() {
+    objectAppear(TEXTCLOCK[0]);
+}
+function clearObservation() {
+    observ.innerHTML = EMPTY;
+}
+function allDisappear() {
+    dotsDisappear();
+    xTextDisappear();
+    yTextDisappear();
+    clockDisappear();
+    outputDisappear();
+    for (const object of OBJECTS) {
+        fillColor(object, "#008000");
+    }
+}
+function outputHandler() {
+    let state = currPos;
+    TEXTOUTPUT[0].textContent = outputStream[state];
+    setter(TEXTOUTPUT[0].textContent, OBJECTS[3]);
+    setter(outputStream[state], INPUTDOTS[2]);
+}
+function changeInput() {
+    let state = currPos;
+    TEXTINPUT[0].textContent = inputStream[0][state];
+    TEXTINPUT[1].textContent = inputStream[1][state];
+    setCoordinates(45, 440, TEXTINPUT[0]);
+    setCoordinates(45, 490, TEXTINPUT[1]);
+    svg.append(TEXTINPUT[0]);
+    svg.append(TEXTINPUT[1]);
+    setActive(state);
+    setter(TEXTINPUT[0].textContent, OBJECTS[0]);
+    setter(TEXTINPUT[1].textContent, OBJECTS[1]);
+    if (inputStream[0][state] === 1) {
+        fillColor(INPUTDOTS[0], "#29e");
+    }
+    else {
+        fillColor(INPUTDOTS[0], "#eeeb22");
+    }
+    if (inputStream[1][state] === 1) {
+        fillColor(INPUTDOTS[1], "#29e");
+    }
+    else {
+        fillColor(INPUTDOTS[1], "#eeeb22");
+    }
+    if (state === 0) {
+        fillColor(INPUTDOTS[2], "#eeeb22");
+    }
+}
+
+function clockToZero() {
+    TEXTCLOCK[0].textContent = 0;
+    svg.appendChild(TEXTCLOCK[0]);
+    setCoordinates(598, 704, TEXTCLOCK[0]);
+    fillColor(OBJECTS[2], "#eeeb22");
+    observ.innerHTML = "Clock set to 0";
+}
+function clockToOne() {
+    TEXTCLOCK[0].textContent = 1;
+    svg.appendChild(TEXTCLOCK[0]);
+    setCoordinates(598, 704, TEXTCLOCK[0]);
+    fillColor(OBJECTS[2], "#29e");
+    observ.innerHTML = "Clock set to 1";
+}
+
+function increaseCurrPos() {
+    currPos++;
+}
+
+function reboot() {
+    for (const elements of ARRAYX) {
+        elements.style.fill = "#29e";
+    }
+    for (const elements of ARRAYY) {
+        elements.style.fill = "#29e";
+    }
+    for (const text of TEXTINPUT) {
+        text.textContent = 2;
+    }
+    for (const text of TEXTCLOCK) {
+        text.textContent = 2;
+    }
+}
+function display() {
+    observ.innerHTML = "Simulation has finished. Press Reset to start again";
+    observ.innerHTML += "<br />";
```

Comment:
Please do not use HTML tags for formatting. HTML is for structure, CSS is for formatting

Reply:
Removed that line